src/pages/dashboard/addon/[addon_id]/index.tsx

```diff
@@ -9,7 +9,7 @@ import { Button } from "@components/input/button.module";
 import { Toggle } from "@components/input/toggle";
 import { User } from "@components/dashboard/user";
 
-import useBoolean from "@components/functions/useboolean";
+import { useBoolean } from "@components/functions/hooks/useBoolean";
```

Comment:
I meant something like `@hooks/useBoolean`, you don't want to put hooks in the same directory as components cuz they're kinda unrelated